core/src/assets/config/config.json

```diff
@@ -60,18 +60,18 @@
       "OBSERVABILITY": {
         "config": {
           "links": [
-            [
-              "Grafana",
-              "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/monitoring-grafana"
-            ],
-            [
-              "Kiali",
-              "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/kiali"
-            ],
-            [
-              "Tracing",
-              "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/tracing"
-            ]
+            {
```

Comment:
what about the `isEnabled` flag? 
I'm thinking if we should have this flag for the whole OBSERVABILITY feature only or should we also flag every single link 🤔